spec/classes/candlepin_spec.rb

```diff
@@ -210,12 +188,15 @@
               with_content(sensitive(/^jpa.config.hibernate.connection.url=.*sslrootcert=\/etc\/ssl\/postgresql.pem/)).
               with_content(sensitive(/^org.quartz.dataSource.myDS.URL=.*sslrootcert=\/etc\/ssl\/postgresql.pem/))
           end
+        end
 
-          it do
-            is_expected.to contain_cpdb_create('candlepin').
-              with_ssl_options("?ssl=true&sslrootcert=/etc/ssl/postgresql.pem")
-            is_expected.to contain_cpdb_update('candlepin').
-              with_ssl_options("?ssl=true&sslrootcert=/etc/ssl/postgresql.pem")
+        context 'with db_manage_on_startup Halt, Report or None' do
```

Comment:
The params below suggest it's only with `Halt`
```suggestion
        context 'with db_manage_on_startup Halt' do
```